modular_skyrat/master_files/code/datums/quirks/negative.dm

```diff
@@ -86,3 +86,15 @@
 	var/datum/quirk/blooddeficiency/synth/bd_synth = new
 	qdel(src)
 	return bd_synth.add_to_holder(new_holder, quirk_transfer)
+
+/datum/quirk/gifted
```

Comment:
comment this out please, its garbage and shouldn't be on bubber

Reply:
Yes I'm aware of it.

Reply:
We're keeping this. I am going upstream later to make it a proper free points anyway